### PR TITLE
Enabled live graphs in device-details page

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.android_sense.device-view/public/js/device-detail.js [hunk 1]
```diff
@@ -38,7 +38,7 @@
     $(document).ready(function(){
         $(".panel-body").removeClass("hidden");
         $("#loading-content").remove();
-        loadOperationBar(deviceType);
+        //loadOperationBar(deviceType);
```
Comment (mohanvive) [inline features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.android_sense.device-view/public/js/device-detail.js]: @charithag , Just wondering if this line does not required then can we please remove it than commenting it..

File: features/device-mgt-iot-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/stats.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
 function onRequest(context) {
-        return context;
+    var log = new Log("stats.js");
+    var operationModule = require("/modules/operation.js").operationModule;
+    var device = context.unit.params.device;
+    log.info(device);
```
Comment (mohanvive) [inline features/device-mgt-iot-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/stats.js]: @charithag , whether this info log is required for each request ?

File: features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.android_sense.device-view/public/js/device-detail.js (hunk 1 above)
Comment (charithag) [inline features/device-mgt-iot-androidsense-feature/org.wso2.carbon.device.mgt.iot.androidsense.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.android_sense.device-view/public/js/device-detail.js]: @mohanvive, Thanks for the mentioning this. This was temporally disabled earlier until the operation bar implementation is completed. But later this file is removed by using a generic js file for all device types.
https://github.com/wso2/carbon-device-mgt/blob/master/components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.view/public/js/device-view.js

File: features/device-mgt-iot-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/stats.js (hunk 2 above)
Comment (charithag) [inline features/device-mgt-iot-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/devicemgt/app/units/iot.unit.device.stats/stats.js]: @mohanvive, This was added by a mistake. Unfortunately this unit is no longer exist today to remove the unnecessary log.

**Pull request merged**